content/applications/hr/fleet/accidents.rst

```diff
@@ -0,0 +1,281 @@
+  measure are either in kilometers (:guilabel:`km`) or miles (:guilabel:`mi`), depending on how the
+  selected vehicle was configured.
+- :guilabel:`NOTES`: enter the specific details of the accident at the bottom of the service form,
+  such as `Hit a deer` or `Rear-ended at an intersection while stopped`.
+
+Odoo provides the ability to attach any important paperwork, such as repair estimates and police
+reports, to the service record. To do so, click the :icon:`fa-paperclip` :guilabel:`(paperclip)`
+icon, located in the *chatter* of the form, and a file explorer pop-up window appears. Navigate to
+the desired record, and click :guilabel:`Open` to upload the file.
+
+  .. note::
+     Once a file is added to a service record, a :guilabel:`Files` section appears in the *chatter*.
+     To attach more records, click :icon:`fa-plus-square` :guilabel:`Attach files` to add more
+     documents.
+
+.. image:: accidents/service-form.png
+   :align: center
+   :alt: Enter the information for an accident repair.
+
+Service stages
+==============
+
+In Odoo's *Fleet* app, there are four default service stages:
+
+.. tabs::
+
+   .. tab:: New
+
+      The default stage when a service record is created. The service has been requested, but
+      repairs have not begun. The :guilabel:`Cost` field for this stage remains zero.
+
+   .. tab:: Running
+
+      The repair is in-process, but not yet complete. The estimate for repairs is listed in the
+      :guilabel:`Cost` field.
+
+   .. tab:: Completed
+
+      All repairs listed on the service form have been completed. The :guilabel:`Cost` field is
+      updated to reflect the final total cost charged for the repairs.
+
+   .. tab:: Cancelled
+
+      The service request has been cancelled.
+
+During the repair process, change the service status to reflect the vehicle's current state in one
+of two ways: on the individual :ref:`service record <fleet/service_record>`, or in the :ref:`Kanban
+service view <fleet/Kanban>`.
+
+.. _fleet/service_record:
+
+Service record
+--------------
+
+Open the main :guilabel:`Services` dashboard, by navigating to :menuselection:`Fleet app --> Fleet
+--> Services`. Next, click on the individual service record to open the detailed service form. Click
+the desired stage in the top-right corner, above the service form, to change the status.
+
+.. image:: accidents/running.png
+   :align: center
+   :alt: The stages as seen from the service form.
+
+.. _fleet/Kanban:
+
+Kanban view
+-----------
+
+Open the main :guilabel:`Services` dashboard, by navigating to :menuselection:`Fleet app --> Fleet
+--> Services`. First, click the :icon:`oi-view-kanban` :guilabel:`Kanban` icon in the top-right of
+the screen, which organizes all repairs by vehicle.
+
+Next, remove the default :guilabel:`Service Type` filter in the :guilabel:`Search...` bar. Upon
+doing so, all services appear in a Kanban view, organized by their respective :guilabel:`Status`.
```

Comment:
Since we're already editing this doc from my review, please make the small update to change "Search..." to "search bar"
```suggestion
Next, remove the default :guilabel:`Service Type` filter in the search bar. Upon doing so, all
services appear in a Kanban view, organized by their respective :guilabel:`Status`.
```